content/en/docs/tasks/job/coarse-parallel-processing-work-queue.md

```diff
@@ -45,10 +45,20 @@ cluster and reuse it for many jobs, as well as for long-running services.
 
 Start RabbitMQ as follows:
 
+Step: 1
 ```shell
 $ kubectl create -f examples/celery-rabbitmq/rabbitmq-service.yaml
```

Comment:
please remove `$` as per recent style guideline changes

Reply:
@DanyC97  I made removed $ can u review the changes.